core/src/main/java/org/opensearch/sql/data/type/WideningTypeRule.java

```diff
@@ -12,25 +11,18 @@
 import org.opensearch.sql.exception.ExpressionEvaluationException;
 
 /**
- * The definition of widening type rule for expression value.
- * ExprType     Widens to data types
- * INTEGER      LONG, FLOAT, DOUBLE
- * LONG         FLOAT, DOUBLE
- * FLOAT        DOUBLE
- * DOUBLE       DOUBLE
- * STRING       STRING
- * BOOLEAN      BOOLEAN
- * ARRAY        ARRAY
- * STRUCT       STRUCT
+ * The definition of widening type rule for expression value. ExprType Widens to data types INTEGER
```

Comment:
This also doesn't make sense now, unfortunately.

Reply:
Thanks! fixed in [d04dd9f](https://github.com/Bit-Quill/opensearch-project-sql/pull/312/commits/d04dd9f04d966cfc1c0e047cccb8bab3e6c386ba)